changelog/fragments/actions-go-e2e-bump.yaml

```diff
@@ -0,0 +1,18 @@
+# entries is a list of entries to include in
```

Comment:
Why do we need a changelog?
How does bump the ci tests affect who uses the SDK CLI tool?

Reply:
Good point. I think I added it without thinking, will remove it. However this PR may get closed and this step be done as part of https://github.com/operator-framework/operator-sdk/pull/5843